packages/ckeditor5-dev-changelog/src/constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+export const CHANGELOG_FILE = 'CHANGELOG.md';
+export const CHANGELOG_HEADER = 'Changelog\n=========';
+export const ORGANISATION_NAMESPACE = '@ckeditor';
```

Comment:
For the monorepo purposes this is fine to hardcode the organisation namespace, as all our monorepo packages are scoped in `@ckeditor/*`. But this for sure will have to be changed later when providing support for generating a changelog for any package.

Reply:
This needs to be changed in this PR. We cannot hardcode any "CKEditor" related things, as the tool will be used also outside the "CKEditor" project.

Reply:
This is now an argument to the main function. Fixed in [f076fb4](https://github.com/ckeditor/ckeditor5-dev/pull/1090/commits/f076fb453f988f8200417362136257017c7f33dd).